### PR TITLE
Added Box missing justify properties.

### PR DESCRIPTION
**Pull request by sofiageo**
Added around and evenly.

### CODE REVIEW

**Review by ericsoderberghp — APPROVED** [review]

**Comment by ericsoderberghp** [comment]
Thanks for your contribution! :)

File: src/types.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const types = {
           align: ['stretch', 'start', 'center', 'end'],
           direction: ['column', 'row', 'row-responsive'],
           gap: ['xsmall', 'small', 'medium', 'large', 'xlarge'],
-          justify: ['between', 'start', 'center', 'end'],
+          justify: ['around', 'between', 'start', 'center', 'end', 'evenly'],
```
Comment (ShimiSun) [inline src/types.js:57]: I'm thinking `stretch` value is also missing, it should be the default, maybe we should replace it with the current `undefined` that is shown on the drop options

Comment (sofiageo) [reply src/types.js:57]: Sorry for not adding it, it is also missing from the Box typescript file so I was not sure about it :)

Comment (ShimiSun) [reply src/types.js:57]: @sofiageo no worries buddy, thanks for the PR!

**Pull request merged**